app/migrations/Version20190319002039.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\Exception\SkipMigration;
+use Mautic\CoreBundle\Doctrine\AbstractMauticMigration;
+use Mautic\PointBundle\Entity\LeadPointLog;
+
+class Version20190319002039 extends AbstractMauticMigration
+{
+    /**
+     * @throws SkipMigration
+     * @throws \Doctrine\DBAL\Schema\SchemaException
+     */
+    public function preUp(Schema $schema): void
+    {
+        $table = $schema->getTable($this->prefix.LeadPointLog::TABLE_NAME);
+
+        if ($table->hasColumn('internal_id')) {
+            throw new SkipMigration('Schema includes this migration');
+        }
+    }
+
+    public function up(Schema $schema): void
+    {
+        $tableName = $this->prefix.LeadPointLog::TABLE_NAME;
+        $this->addSql("ALTER TABLE {$tableName} ADD internal_id BIGINT UNSIGNED NOT NULL");
```

Comment:
Note to self:

I tested this query on a table with some records. It did not error out but set the value to 0 even though there is no default value defined for this column.